internal/cli/atlas/security/ldap/save.go

```diff
@@ -127,6 +127,11 @@ func SaveBuilder() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "save",
 		Short: "Save an LDAP configuration.",
+		Example: fmt.Sprintf(` The following example configures an LDAP server to authenticate and authorize MongoDB users.
+  %s security ldap save --authenticationEnabled true --authorizationEnabled true 
+  --hostname atlas-ldaps-01.ldap.myteam.com --bindUsername 
+  "CN=Administrator,CN=Users,DC=atlas-ldaps-01,DC=myteam,DC=com" 
+  --bindPassword changeMe`, cli.ExampleAtlasEntryPoint()),
```

Comment:
no need to provide `true` with boolean flags
```suggestion
  %s security ldap save --authenticationEnabled --authorizationEnabled 
  --hostname atlas-ldaps-01.ldap.myteam.com --bindUsername 
  "CN=Administrator,CN=Users,DC=atlas-ldaps-01,DC=myteam,DC=com" 
  --bindPassword changeMe`, cli.ExampleAtlasEntryPoint()),
```